site/assets/scss/_component-examples.scss

```diff
@@ -330,6 +330,11 @@
     white-space: pre;
     background-color: transparent;
     border: 0;
+
+    // Undo tabindex that's automatically added by Hugo
+    &:focus {
+      outline: 0;
```

Comment:
Consider using `outline-color: transparent` instead: that'd allow OS' high constrasts modes to better operate.

Reply:
We use `outline: 0` in around a dozen other spots, maybe we need a more holistic look at that? Going to keep as-is because of that for now.

Reply:
Seems legit, we'll address that everywhere at once (probably after your global focus vars PR).